Detectors/CTP/workflow/src/RawDecoderSpec.cxx

```diff
@@ -135,6 +137,13 @@ void RawDecoderSpec::run(framework::ProcessingContext& ctx)
   if (fatal_flag) {
     ret = mDecoder.decodeRawFatal(inputs, filter);
   } else {
+    if (mDecodeinputs) {
+      const auto ctpcfg = inputs.get<o2::ctp::CTPConfiguration*>("ctpconfig");
+      if (ctpcfg != nullptr) {
+        mDecoder.setCTPConfig(*ctpcfg);
+      }
+    }
```

Comment:
This should be done only once in the beginning of the run processing, right? Why don't you do thin in after https://github.com/AliceO2Group/AliceO2/pull/13682/files#diff-730222aede43b6dd828e45b24b01ba97b3d88fc8c073a79837364b2fb9408b65R237 ?